ilearn/generate-onderwijsstructuur/input/onderwijsstructuur.addendum.ttl

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
I would make this valid Turtle, normally concatenating turtle strings doesn't give issues, but then this file is valid and self-standing

Reply:
Done, see https://github.com/i-Learn-SKOS/common-conceptschemes/pull/7/commits/a7abf3c19d83c2643d35405b10b761aef109e58f